modules/core/types/events.go

```diff
@@ -0,0 +1,29 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const ErrorAttributeKeyPrefix = "ibccallbackerror-"
+
+// ConvertToErrorEvents converts all events to error events by appending the
+// error attribute prefix to each event's attribute key.
+func ConvertToErrorEvents(events sdk.Events) sdk.Events {
```

Comment:
Can we make this more explicit to our use case? I don't really want to encourage reusing this short term solution. I would be in favour of keeping private and moving to the file it is used

Reply:
made private and moved to the file it is used in (`msg_server.go`)